llvmlite/tests/test_ir.py

```diff
@@ -234,6 +235,34 @@ def test_metadata_null(self):
         self.assertInText("!{ i32* null }", str(mod))
         self.assert_valid_ir(mod)
 
+    def test_debug_info(self):
+        def order(d):
+            pairs = [(k, d[k]) for k in d]
+            pairs.sort()
+            return OrderedDict(pairs)
```

Comment:
The most ergononic API for `add_debug_info` is accepting a dict, yet the hash randomization screws up tests, so this workaround is used.